### PR TITLE
Feature/wapp promo clipboard

### PR DESCRIPTION
**Pull request by SanderHageman**
Add a button to the 'description'-box of the admin interface for activities. The copied text can then be pasted as promo in WhatsApp for convenience.

```
*Connecting | zondag 10 december 14:12 - 19:14 | Locatie: Goldman's Pharmacy | €15.31*

https://koala.svsticky.nl/activities/21

Architecto et sapiente. Quis non molestias. Qui aut ea. Sit similique adipisci. Maxime voluptas tempora.
```

image

### CODE REVIEW

File: app/javascript/src/admin/activities.js [hunk 1]
```diff
@@ -445,6 +448,14 @@ function addCopyTableCallBack() {
...
 }
 
+function copyPromoToClipboard() {
+  new Clipboard(".btn-clipboard-wapp", {
+    text: function (trigger) {
+      return trigger.getAttribute("data-clipboard-text");
+    },
+  });
+}
+
```
Comment (SanderHageman) [inline app/javascript/src/admin/activities.js:458]: Only interesting part of the `.js`-files, the rest was the linter adding some commas

**Review by SilasPeters — DISMISSED** [review]
These changes seem harmless. I have yet to test it locally, but I assume you already did. In that case, I believe this can be merged.

**Review by HugoPeters1024 — DISMISSED** [review]
Sander said it worked on his machine, LGTM!

**Review by SilasPeters — APPROVED** [review]
It did not work on my machine, but Hugo the haskell god knows best.

**Pull request merged**